webapp/app/__init__.py

```diff
@@ -32,6 +33,9 @@
     strategy='moving-window'
 )
 
+flask_static_digest = FlaskStaticDigest()
+flask_static_digest.init_app(app)
```

Comment:
I guess that this will change after the merge of the app factory?

Reply:
Correct.